### ISSUE TITLE
[BUG] Downloading files dont have an extension

### ISSUE THREAD

**Issue by IamIndeedGamingAsHardAsICan03489** [pytube/pytube#1121]
Hi so, i tried creating a small script that downloads the audio of the video only,
here is the basic idea
`yt = YouTube(links[i])
stream = yt.streams.get_by_itag(140)
print(stream)
print(stream.download(path, yt.title))`
I loop over inputted links, check if they are valid and proceed with the stuff

Everythings working great, apart from the file not having an extension of mp3, i tried just adding .mp3 at the end, but it ends up with a very bad quality and an error when trying to decode the audio in a video editing software like premiere pro, 
im running Python 3.9.7
with Pytube version 11.0.1

**Comment by IamIndeedGamingAsHardAsICan03489** [comment]
![Screenshot_3](https://user-images.githubusercontent.com/81194357/133944332-256072e8-a0ef-4583-ba33-3a76c23d1b84.png)
Here is how an example downloaded file looks like (extenstions are visible)

**Comment by chessboards** [comment]
If you add the extension to the filename, it will allow you to open it. 

For instance `stream.download(path, f"{yt.title}.mp4")`

**Comment by IamIndeedGamingAsHardAsICan03489** [comment]
> If you add the extension to the filename, it will allow you to open it.
> 
> For instance `stream.download(path, f"{yt.title}.mp4")`

yes, i am already tried doing that, when i pop the file into premire pro it cannot be decoded due to not having the bit rate set
